os/tools/mkbootparam.py

```diff
@@ -103,12 +109,22 @@ def make_bootparam():
 
     with open(bootparam_file_path, 'wb') as fp:
         # Write Data
+        bootparam_version = 0
+        bootparam_format_version = 10 #v1.0
```

Comment:
Is there any special reason to have different type of version between `bootparam_version` and `bootparam_format_version`?
And if you want to use float type version, then how about using float type? It is bad to understand that `10` means `v1.0`.